### DISCUSSION TITLE
importing code into Phoenix

### DISCUSSION THREAD

**Discussion by DougLassiter** [phcode-dev/phoenix#1738] [General]
I'm trying to use the Phcode editor, and I can't understand how to input an existing html page and make it a "current project". I can make it a "working file", but I can't get it rendered unless it is a real "project". Help!

**Comment by abose** [comment]
A project is just a folder in phcode. To open a folder, you can either drag the folder containing the html file to phcode or select file - open folder.

**Comment by DougLassiter** [comment]
Thanks. When I drag the html file to phcode, it ends up in "Working Files", and is not considered a "Project". When I "Start Project", I can only open a folder, and not an html file. Now, that being said, if I put that html file in a folder, I CAN open that folder and make it a project. Yay!

So now that I have a Preview rendered image of the html file, how do I edit that Preview? That is, I'm assuming that WYSIWYG ought to let me edit the rendered image. Is that not the case?

**Reply by abose** [reply]
Phcode is not wysiwyg efitor for html. The code changes can be live previewed, but you cannot directly edit the live preview.

**Reply by DougLassiter** [reply]
That's useful to know. https://mailchimp.com/resources/wysiwyg-html-editor says different, at least for Brackets, which I understand was the predecessor of Phcode.

**Reply by abose** [reply]
Brackets is not WYSIWYG too and works the same as phcode. We may add it in a few releases this year in phcode.
